### DISCUSSION TITLE
Multiple plugins

### DISCUSSION THREAD

**Discussion by a3pelawi** [tauri-apps/tauri#4275] [Q&A] [answered]
I want to create a complete desktop application using tauri. for management for the backend I plan to group it in several plugins, such as database, security and others. my question, how to insert multiple plugins with tauri.

**Comment by FabianLars** [comment, marked as answer]
If you're talking about these plugins: https://tauri.studio/v1/guides/features/plugin you should be able to just call the `.plugin()` method multiple times, like:
```rs
fn main() {
  tauri::Builder::default()
    .plugin(my_awesome_plugin::init())
    .plugin(database_plugin::init())
    .plugin(some_other_plugin::init())
    .run(tauri::generate_context!())
    .expect("failed to run app");
}
```

**Reply by a3pelawi** [reply]
Thanx @FabianLars 
I will try it 😊🙏
